src/ods-button.js

```diff
@@ -67,15 +69,19 @@ class OdsButton extends LitElement {
     return disabled || isactive;
   }
 
+  buttonCallback() {
```

Comment:
This is a default callback within the custom element as to alert the consumer of the component if they have not correctly bound an event to the custom element correctly. 

When the `buttonCallback` prop is overwritten by an external function, this alert will be dismissed.